### ISSUE TITLE
Reclaim tool: Is the "Proof transaction is pending" staying forever a bug? Is the "Reattach complete" enough?

### ISSUE THREAD

**Issue by Christo81** [iotaledger/wallet#682]
I have been trying to move my iota funds from an old seed (pre October) to a new using the 2.5.4 iota for Windows interface. I have been pressing the "Reattach" button for hours but the status doesn't change, remains "Proof transaction is pending". I am really WORRIED about my funds, since I can't even send anything from my old seed, it is pending for days. Any help would be deeply appreciated.

**Comment by Alino** [comment]
I have been pressing reattach and rebroadcast on all different nodes for 3 days and it doesn't help. Our iota is stuck

**Comment by Christo81** [comment]
So we should just wait till the network calms down or it is upgraded? We don't have to perform any other actions? I think the funds will not be transferred to the new seed unless the proof transaction is verified, which means we might have to do the reclaim tool again when the network is working.

**Comment by RomanPushka30** [comment]
Same problem. Decisions are nowhere to be found.

**Comment by Iamamillionaire** [comment]
yep, same here.. more than 2 days and still pending.. sent 2 small transactions to 2 different wallets - both stuck!

**Comment by masonjeffers** [comment]
Same problem here as well.

**Comment by ThomasPepperz** [comment]
Try different nodes, if minWeight is 14 try 15 or vice a versa. Also, try
reattaching frequently, generating receive addresses, and logging in and
out of the wallet. If you go to slack and find the support channel or the
node channel you may be able to find a node that is processing quickly.

**Comment by Iamamillionaire** [comment]
will try changing the weight to 15.. already been trying reattaching and changing other nodes, no help so far..  would you send an invite link to slack?

**Comment by ThomasPepperz** [comment]
https://slack.iota.org

Here you go, friends. Try (I don't know how) to keep it out of the hands of
the trolls.

**Comment by Iamamillionaire** [comment]
I already tried that website - it's by invite only..

**Comment by Alino** [comment]
I tried minWeight 14 and 15
I tried rebroadcasting every half hour
I tried reattaching every half hour
I tried rebroadcasting and reattaching every half hour on different nodes
I was on slack - this is the correct url https://iotatangle.slack.com/
On slack they told me the same things, to reattach and to try different nodes and changes minWeight to 14 or 15.
They gave me this link on slack to try these nodes - https://iotasalad.org/nodes
I tried these nodes, and reattach on them and no results.
I have like 30 reattaches and my transaction is unconfirmed for4 days.

**Comment by funkyfranz** [comment]
I am experiencing similar issues. 
Transactions don't go through anymore since 3 days.
2 days ago maybe there was an attack on the network. I see over 100 transactions in my transaction history, all of them transferring the total sum of iota away from my wallet. None of these transactions are confirmed. However, my wallet shows 0 iota left.
I tried creating a couple of new addresses, nothing changed.
So I created a new seed, and used the reclaim tool of the Windows client. After a couple of failed attempts it gave me a success notice.
When I open the new wallet, however, it shows 0 balance, and 0 transactions.

Would be great to hear from the developer team whether this is a known issue.

**Comment by funkyfranz** [comment]
This is a working invite link to the slack group:
https://join.slack.com/t/iotatangle/shared_invite/enQtMjg0ODgwNjgwMTM0LTdjOTg4MmRjNmE2ZWQwMDE3OWVkYjZkZGVmODA2ZDA0MDlkNzkxZjIyNTE2M2VhZGJlYThiZjlkYTU5ZTVhMzY

And on the slack group I found a link with some information about the reclaim status: https://forum.helloiota.com/1242/Reclaim-Status
